### DISCUSSION TITLE
How can we change the color used in case of error?

### DISCUSSION THREAD

**Discussion by mlarcher** [mantinedev/mantine#1902] [Q&A]
I tried to target `.mantine-Select-error, .mantine-TextInput-error, .mantine-NumberInput-error` from globalStyles, but it only affects the error text, not the border/icon/cursor colors. 

Also, it would be nice to be able to change that color for all Mantine form inputs at once. 

What is the correct way to change the error color of form inputs globally ?

**Comment by rtivital** [comment]
You can use `.mantine-Input-invalid` to customize styles of all inputs on MantineProvider – https://mantine.dev/styles/styles-api/#inputs-styles-api-on-mantineprovider

**Reply by mlarcher** [reply]
Thank you.
Styles on MantineProvider looks cleaner than using globalStyles. Is it possible to go one step further and have a keyword instead of a className to target `.mantine-Input-invalid` and `.mantine-InputWrapper-error` ? It seems InputWrapper is not documented anymore, so it's hard to tell...

**Reply by rtivital** [reply]
Input.Wrapper component documentation is on Input page – https://mantine.dev/core/input/

**Reply by mlarcher** [reply]
Perfect. Thank you !
